java/.gitignore

```diff
@@ -1,4 +1,9 @@
-# Note: this file is a copy of https://github.com/github/gitignore/blob/main/Gradle.gitignore
+# Ignore gradlew files
+gradle/wrapper/
+gradlew
+gradlew.bat
+
+# Note: this rest of the file is a copy of https://github.com/github/gitignore/blob/main/Gradle.gitignore
```

Comment:
I thought this was already done in #711 ? Why is it showing up here as well?

Reply:
github updated the base branch automatically but I think I had to make a change to the earlier commits so I will need to rebase this branch